defopt.py

```diff
@@ -312,8 +319,28 @@ def _parse_doc(func):
     return _Doc(doctext, tuples)
 
 
+def _itertext_ansi(node):
+    # Implementation modified from `ElementTree.Element.itertext`.
+    tag = node.tag
+    if not isinstance(tag, str) and tag is not None:
```

Comment:
This is `basestring` in the Python 2 version of `itertext`. We should account for that.